.gitignore

```diff
@@ -1,12 +1,10 @@
-.DS_Store
-node_modules
-TODO.md
-lib
-docs/.vuepress/dist
-examples/**/build.js
-types/typings
-types/test/*.js
-explorations
+/dist
```

Comment:
I guess we need to add `!/dist/logger.d.ts` ?

Reply:
Ah, good point! Yeah I'll add that. `logger.d.ts` will be tracked regardless of it because it is already committed, but it's nice to have precise definition 👍

Reply:
And fixed 👍